Utilities/File.cpp

```diff
@@ -229,46 +232,20 @@ bool fs::create_dir(const std::string& dir)
 
 bool fs::create_path(const std::string& path)
 {
-	g_tls_error = fse::ok;
```

Comment:
Because I just rewrote the whole method body without realizing there would
be side effects. Sorry, I'll add it back.

On Wed, Dec 23, 2015 at 11:27 PM, Raul Tambre <notifications@github.com>
wrote:

> In Utilities/File.cpp
> <https://github.com/RPCS3/rpcs3/pull/1371#discussion_r48400085>:
>
> > @@ -229,46 +232,20 @@ bool fs::create_dir(const std::string& dir)
> >
> >  bool fs::create_path(const std::string& path)
> >  {
> > -	g_tls_error = fse::ok;
>
> Why remove this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RPCS3/rpcs3/pull/1371/files#r48400085>.
>